src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/extra_order_button_actions.html.twig

```diff
@@ -0,0 +1,90 @@
+{#**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{# Rendering of hook actionGetAdminOrderButtons, see ActionsBarButtonsCollection #}
+{% if (backOfficeOrderButtons.isEmpty() != true) %}
```

Comment:
Prefer strict operator maybe ? Like `!==`, I don't know, if isEmpty isn't returning strict value, then I'm wrong

Reply:
I don't think strict operators exist in twig

Reply:
I confirm, there is no Twig equivalent fro `!==` although it seems there is one for `===` which is `same as`
https://www.branchcms.com/learn/docs/developer/twig/operators